members/S001194.yaml

```diff
@@ -189,15 +178,12 @@ contact_form:
             - Welfare
             - "Women's Issues"
     - click_on:
-        - selector: input#response_y
-    - wait:
-        - value: 1
+        - selector: "label[for='input-3048820B-A361-6166-70F4-2D74C1E208F4']"
     - click_on:
-        - value: Send
-          selector: input#side-search-btn
+        - selector: "div > div > button[type='submit']"
     - find:
-        - selector: "h2"
-          value: "Thank You"
+        - selector: "h1"
+          value: "Mahalo"
```

Comment:
`find` step doesn't actually support a `value` property, so not sure why this was here to begin with. this find selector is super vague though, is there a stronger one we could use?

Reply:
Sadly no, but it is (for now) unique to the thank you page